### ISSUE TITLE
Training repetition

### ISSUE THREAD

**Issue by f1yfisher** [facebookresearch/UmeTrack#11]
I have tried many ways to reproduce your method, but the results always much different from those in the paper. For full model with the pose loss, the results are 19.214mm for known hands and 19.756mm for unknown hands. Can you provide more details for training? 
1.  Did you adopt some learning schedulers during training? 
2. Could you provide sepcific hyper-parameters in data augmentation? 
3. Whether you use only 2-view images for training? 
4. Needn't you compute the mean of the first item of the pose loss in eq4?
5. Did you direct sum the pose loss of known hands and unknown hands without any hyper-parameters?

**Comment by guker** [comment]
do you just use umetrack_data to train model?

**Comment by f1yfisher** [comment]
> do you just use umetrack_data to train model?

Yes, I use the real and synthetic training data of umetrak_data for training.

**Comment by zhanxiaopan** [comment]
Hi @f1yfisher @guker，
Could you share or open source you training python script.  Since, we now try to re-train the model too. Maybe we can help with each other if theauthor has no plans to open source the training code.

email:  panzi2017@gmail.com

**Comment by guker** [comment]
can you provide training code for me？ thanks very much
email: huqunwei2607@gmail.com

**Comment by edgar-explorer** [comment]
can you provide training code for me？ thanks very much
email: chen1chang2@gmail.com
